frontend/src/features/tinymce/components/Editor.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Spinner } from '@chakra-ui/react'
+import { Editor as TinymceEditor } from '@tinymce/tinymce-react'
+
+import { useTinymceApiKey } from '../hooks/tinymce.hooks'
+
+interface LetterViewerProps {
+  html: string | undefined
+  isLoading: boolean
+  isDisabled?: boolean
+  isInline?: boolean
+}
+
+export const Editor = ({
+  html,
+  isLoading,
+  isDisabled = false,
+  isInline = false,
+}: LetterViewerProps): JSX.Element => {
+  const { tinymceApiKey, isLoadingTinymceApiKey } = useTinymceApiKey()
+  if (isLoading || !html) {
+    return <Spinner />
+  }
+  // tinymce not enabled
+  if (!isLoadingTinymceApiKey && !tinymceApiKey) {
```

Comment:
currently, if the letter has been loaded (`isLoading` is false) but the tinymce api key is not loaded yet (`isLoadingTinymceApiKey` is true), it defaults towards loading the `TinymceEditor` instead of the `Spinner`, would the following change be helpful?

```suggestion
  if (isLoading || isLoadingTinymceApiKey || !html) {
    return <Spinner />
  }
  // tinymce not enabled
  if (!tinymceApiKey) {
```